### ISSUE TITLE
verbose kwarg in cross_val_multiscore()

### ISSUE THREAD

**Issue by hoechenberger** [mne-tools/mne-python#10732]
The signature of `mne.decoding.cross_val_multiscore()` currently looks like this:

```python
@fill_doc
def cross_val_multiscore(estimator, X, y=None, groups=None, scoring=None,
                         cv=None, n_jobs=None, verbose=0, fit_params=None,
                         pre_dispatch='2*n_jobs'):
```
Why aren't we using `verbose=None` and the `@verbose` decorator here, as in all (?) other places?

**Comment by larsoner** [comment]
Not sure if there is any reason not to, feel free to try adding it

**Comment by hoechenberger** [comment]
Closed via #10733
